src/main/java/gov/usgs/wma/waterdata/groundwater/RdbWriter.java

```diff
@@ -130,16 +130,17 @@ protected RdbWriter writeRow(List<String> columns) {
 	 * entry length. This writes only the given length value
 	 * @param length max number of chars to write
 	 * @param value characters to write, non-string values must be converted.
+	 * @return The trimmed value with nulls converted to empty strings.
 	 */
 	protected String validateValue(int length, String value) {
 		// Trim the value to the proper field length.
-		String trimmedValue = value;
+
 		if (value == null) {
 			value = "";
 		}
 		if (value.length() > length) {
-			trimmedValue = value.substring(0, length);
+			value = value.substring(0, length);
 		}
-		return trimmedValue;
+		return value;
```

Comment:
This tricky bit of code actually let null values thru (even though it didn't look like it)